cdc/kv/matcher.go

```diff
@@ -35,7 +36,14 @@ func (m *matcher) matchRow(row *cdcpb.Event_Row) ([]byte, error) {
 		return value, nil
 	}
 	return nil, errors.NotFoundf("prewrite row, startTs:%d", row.GetStartTs())
```

Comment:
maybe we can return a bool indicates whether the match was successful

Reply:
addressed